<launch>
  <node pkg="tf" type="static_transform_publisher" name="tf_camera" args="0 0 -0.55 0 0 0 world camera 100" />
  <node name="fire_nav" pkg="fire_nav" type="fire_nav_node" output="screen">
  <rosparam file="$(find fire_nav)/config/configuration.yaml" command="load" />
  </node>
</launch>